packages/core/src/app/customer/Customer.tsx

```diff
@@ -405,7 +405,9 @@ class Customer extends Component<CustomerProps & WithCheckoutCustomerProps & Ana
 
             const customer = data.getCustomer();
 
-            if (customer && customer.shouldEncourageSignIn && customer.isGuest && !customer.isStripeLinkAuthenticated) {
+            const paymentProviderCustomer = data.getPaymentProviderCustomer();
+
+            if (customer && customer.shouldEncourageSignIn && customer.isGuest && !paymentProviderCustomer?.stripeLinkAuthenticationState) {
```

Comment:
I changed it to `stripeLinkAuthenticationState` as it was needed. After pulling newest changes everything is working fine.